tests/cmds/test_cases.py

```diff
@@ -209,8 +202,12 @@ def test_show_with_include_file_events_calls_file_events_get_all_with_expected_p
     cli_state.sdk.cases.file_events.get_all.assert_called_once_with(1)
 
 
-def test_show_when_py42_raises_exception_prints_error_message(runner, cli_state, error):
-    cli_state.sdk.cases.file_events.get_all.side_effect = Py42NotFoundError(error)
+def test_show_when_py42_raises_exception_prints_error_message(
```

Comment:
For `CASE_DETAILS`, can we make that the actual response for getting a case? And maybe rename it to `CASE_DETAILS_RESPONSE_TEXT` or something similar